python/composio/tools/local/codeanalysis/chunker.py

```diff
@@ -224,37 +222,12 @@ class Chunking:
         language (Language): The loaded Python language for tree-sitter parsing.
 
     Methods:
```

Comment:
The class docstring still mentions removed methods `_setup_tree_sitter()` and `_load_language()`. Please update the docstring to reflect the current implementation.